views/dashboard.handlebars

```diff
@@ -4,7 +4,6 @@
 }}
 
 {{! Required for jupyter-js-widgets }}
-<link rel='stylesheet' href='/components/bootstrap/css/bootstrap.min.css'>
```

Comment:
Since we are no longer using this, you can remove lines 92-94 in **gulpfile.js**.  With that change, feel free to merge.